clients/cloud/confluent-cli/confluent-cli-example.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+topic_name=test1
+
+ccloud topic create $topic_name
+
+num_messages=10
+for ((i=0;i<$num_messages;i++)); do
+  echo "alice,{\"count\":${i}}" | confluent produce $topic_name --cloud --property parse.key=true --property key.separator=,
```

Comment:
this instantiates a client per message, which is expensive, wrap the loop as so instead:
```bash
(for ((i=0;i<$num_messages;i++)); do
  echo "alice,{\"count\":${i}}" ;
done) | confluent produce $topic_name --cloud --property parse.key=true --property key.separator=,
```

Reply:
I refactored this a touch (for printing nicely), but kept the num messages as a var.